src/training/tesstrain_utils.sh

```diff
@@ -164,6 +165,9 @@ parse_flags() {
             --my_boxtiff_dir)
                 parse_value "MY_BOXTIFF_DIR" ${ARGV[$j]:-}
                 i=$j ;;
+            --distort_image)
```

Comment:
Isn't `--distort_image` a boolean argument? Then it should not expect an additional value and simply assign `DISTORT_IMAGE=true`.

Reply:
tesstrain.sh uses 0 and 1 for all the boolean arguments so I have changed it to `DISTORT_IMAGE=1`.

Reply:
Should all the booleans be changed to use `=true` and `=false` rather than `=1` and `=0`?

```
SAVE_BOX_TIFF=0
DISTORT_IMAGE=0
OVERWRITE=0
LINEDATA=0
RUN_SHAPE_CLUSTERING=0
EXTRACT_FONT_PROPERTIES=1
```

Reply:
`=true` and `=false` is more clear than `=1` and `=0` (which can somebody unterstand as integer and not boolean )

Reply:
Done.

Also sorted the list of arguments alphabetically and changed the following defaults to match those required for LSTM training.

```
EXTRACT_FONT_PROPERTIES=false
LINEDATA=true
```

Reply:
Codacy error:

LINEDATA appears unused. Verify use (or export if used externally).
LINEDATA=true ;;

It is used in tesstrain.sh 

https://github.com/tesseract-ocr/tesseract/blob/master/src/training/tesstrain.sh#L84